### DISCUSSION TITLE
problem setting spotlight height

### DISCUSSION THREAD

**Discussion by peerjuettner** [gilbarbara/react-joyride#654] [Q&A] [answered]
Hello
Im trying to hide the spotlight in a perticular instance. 
I've found out that setting the spotlight element's height to 0 would do that for me. However, when I try to set it like so:
`    styles: {
    spotlight: {height: 0  },
    },`
my values aren't applied. 
Please advise on how to set the height manually. 
I'm using joyride@2.3.0

**Comment by gilbarbara** [comment, marked as answer]
hey @peerjuettner 

The spotlight dimensions are calculated dynamically, so you can't change them with `styles`.
Anyway, this library isn't designed to work without it, so there's no recommended way to hide it.
Maybe with `visibility`?

**Reply by peerjuettner** [reply]
I though as much - sadly it's a requirement on a project. I did find out how to do it - I set the background on the spotlight the same as my overlay - it's not the most beautiful way but it works
